app/models/repository.rb

```diff
@@ -357,7 +352,6 @@ def self.update_from_hook(uuid, sender_id)
     repository = Repository.where(host_type: "GitHub").find_by_uuid(uuid)
     user = Identity.where("provider ILIKE ?", "github%").where(uid: sender_id).first.try(:user)
     if user.present? && repository.present?
-      repository.download_metadata(user.token)
```

Comment:
`download_metadata` actually gets called in the `update_all_info_async` hook on the next line, so Libraries was fetching manifests twice when it received a hook 🤦🏻